recipe/meta.yaml

```diff
@@ -31,9 +33,10 @@ requirements:
     - pip
     - setuptools
     - wheel
-    - libqdldl {{ version }}
+    - libqdldl 0.1.7
     - numpy {{ numpy  }}
-    - pybind11 2.10.1
+    - pybind11 2.12.0
+    - pybind11-abi
```

Comment:
is this a mutex? does it need to be put everywhere that pybind is used?
also it's only built for linux-64 AFAICT, not sure why the other platforms are passing

Reply:
pybind11-abi is noarch. It is indeed a mutex, which should be used everywhere pybind11 is used. It didn't matter much until recently, as the abi was at version 4 for an extended time. However now both abi 4 and 5 exist.

Reply:
The ABI is mutually exclusive anyway, though no? i.e. 5 on win or py >= 3.12, 4 otherwise?

Reply:
and even if they weren't, not sure why a mutex would be needed as you'd have two package variants, which conda wouldn't install together? AFAIU a mutex is more like, there are two different packages with different names and we only want one of them at a time